app/_gateway_entities/plugin.md

```diff
@@ -42,6 +43,73 @@ The set of plugins you have access to depends on your license tier.
 
 You can also develop custom plugins, adding your own custom functionality to {{site.base_gateway}}.
```

Comment:
The only thing I can think of that might be nice to add here is a small list of the main plugin "themes" Kong has, just to give an idea what a plugin can do. Like:

- Traffic control
- Authentication
- AI
- etc.